### ISSUE TITLE
Votes Fail to Forward for NuVotifier

### ISSUE THREAD

**Issue by Sully231** [NuVotifier/NuVotifier#58]
My server runs SpongeAPI Version: 7.0.0-snapshot

I've linked NuVotifier to servers, but when you vote on them you get this in the server logs.

![log screenshot for nuvotifier](https://user-images.githubusercontent.com/35669679/35202055-2189a9de-feee-11e7-8eab-fee854357e38.PNG)

**Comment by morrissimo** [comment]
seeing the same thing
```21.01 23:18:58 [Server] Server thread/ERROR [Sponge]: The Scheduler tried to run the task nuvotifier-S-14 owned by Plugin{id=nuvotifier, name=NuVotifier, version=2.3.5, source=/./plugins/nuvotifier-universal-2.3.6.jar}, but an error occured.
21.01 23:18:58 [Server] INFO java.lang.NoSuchMethodError: org.spongepowered.api.Sponge.getCauseStackManager()Lorg/spongepowered/api/event/CauseStackManager;
21.01 23:18:58 [Server] INFO at com.vexsoftware.votifier.sponge.VotifierPlugin$3.run(VotifierPlugin.java:347) ~[VotifierPlugin$3.class:?]
21.01 23:18:58 [Server] INFO at org.spongepowered.api.scheduler.Task$Builder.lambda$execute$0(Task.java:139) ~[Task$Builder.class:1.10.2-2477-5.2.0-BETA-2731]
21.01 23:18:58 [Server] INFO at org.spongepowered.common.scheduler.SchedulerBase.lambda$startTask$0(SchedulerBase.java:183) ~[SchedulerBase.class:1.10.2-2477-5.2.0-BETA-2731]
21.01 23:18:58 [Server] INFO at org.spongepowered.common.scheduler.SyncScheduler.executeTaskRunnable(SyncScheduler.java:81) ~[SyncScheduler.class:1.10.2-2477-5.2.0-BETA-2731]
21.01 23:18:58 [Server] INFO at org.spongepowered.common.scheduler.SchedulerBase.startTask(SchedulerBase.java:179) ~[SchedulerBase.class:1.10.2-2477-5.2.0-BETA-2731]
21.01 23:18:58 [Server] INFO at org.spongepowered.common.scheduler.SchedulerBase.processTask(SchedulerBase.java:165) ~[SchedulerBase.class:1.10.2-2477-5.2.0-BETA-2731]
21.01 23:18:58 [Server] INFO at java.util.concurrent.ConcurrentHashMap$ValuesView.forEach(ConcurrentHashMap.java:4707) [?:1.8.0_152]
21.01 23:18:58 [Server] INFO at org.spongepowered.common.scheduler.SchedulerBase.runTick(SchedulerBase.java:108) [SchedulerBase.class:1.10.2-2477-5.2.0-BETA-2731]
21.01 23:18:58 [Server] INFO at org.spongepowered.common.scheduler.SyncScheduler.tick(SyncScheduler.java:51) [SyncScheduler.class:1.10.2-2477-5.2.0-BETA-2731]
21.01 23:18:58 [Server] INFO at org.spongepowered.common.scheduler.SpongeScheduler.tickSyncScheduler(SpongeScheduler.java:191) [SpongeScheduler.class:1.10.2-2477-5.2.0-BETA-2731]
21.01 23:18:58 [Server] INFO at org.spongepowered.mod.SpongeMod.onTick(SpongeMod.java:271) [SpongeMod.class:1.10.2-2477-5.2.0-BETA-2731]
21.01 23:18:58 [Server] INFO at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_30_SpongeMod_onTick_ServerTickEvent.invoke(.dynamic) [?:?]
21.01 23:18:58 [Server] INFO at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) [ASMEventHandler.class:?]
21.01 23:18:58 [Server] INFO at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:632) [EventBus.class:?]
21.01 23:18:58 [Server] INFO at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:588) [EventBus.class:?]
21.01 23:18:58 [Server] INFO at net.minecraftforge.fml.common.FMLCommonHandler.onPreServerTick(FMLCommonHandler.java:274) [FMLCommonHandler.class:?]
21.01 23:18:58 [Server] INFO at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:602) [MinecraftServer.class:?]
21.01 23:18:58 [Server] INFO at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471) [MinecraftServer.class:?]
21.01 23:18:58 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_152]```

versions:
* `nuvotifier-universal-2.3.6.jar`
* `spongeforge-1.10.2-2477-5.2.0-BETA-2731.jar`

**Comment by Ichbinjoe** [comment]
https://github.com/NuVotifier/NuVotifier/commit/28b4d8dcd0923716a582ff54e49cfdd92383fd42

This was fixed as part of 2.3.6. Please update to 2.3.6, and reopen if you see this again.
